Loop/Managers/LocalTestingScenariosManager.swift

```diff
@@ -30,6 +30,10 @@ final class LocalTestingScenariosManager: TestingScenariosManagerRequirements, D
             delegate?.testingScenariosManager(self, didUpdateScenarioURLs: scenarioURLs)
         }
     }
+    
+    var pluginManager: PluginManager {
+        deviceManager.pluginManager
```

Comment:
The `LocalTestingScenariosManager` is only instantiated from within the `DeviceDataManager` which already has reference to the `pluginManager`. `LoopAppManager` is not responsible for creating this `LocalTestingScenariosManager`.

Reply:
This would snowball quite a bit. The `LocalTestingScenariosManager` is using quite a bit of functionality from the `deviceManager`. I'm going to hold off on this for now.